commit 1de3257b6516be198627ca957aa58601bab141b6
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Thu May 12 20:19:33 2022 +0900

    npm set-script server-generate "graphql-codegen --config codegen.yml --watch ./schema.gql"

diff --git a/server/package.json b/server/package.json
index d93697d..ae66477 100644
--- a/server/package.json
+++ b/server/package.json
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "server-start": "ts-node-dev --watch schema.gql,data.json --respawn index.ts"
+    "server-start": "ts-node-dev --watch schema.gql,data.json --respawn index.ts",
+    "server-generate": "graphql-codegen --config codegen.yml --watch ./schema.gql"
   },
   "keywords": [],
   "author": "",
